RecoTracker/LSTCore/standalone/README.md

```diff
@@ -106,186 +170,44 @@ Comparing two different runs
 
-``` bash
-git clone --filter=blob:none --no-checkout --depth 1 --sparse --branch CMSSW_14_1_X https://github.com/cms-sw/cmssw.git
-cd cmssw
-git sparse-checkout add HeterogeneousCore/AlpakaInterface
-git checkout
 ```
-
-Then all that is left to do is set some environment variables. We give an example of how to do this in lnx7188/cgpu-1.
-
-```bash
-# These two lines are only needed to set the right version of gcc and nvcc. They are not needed for standard installations.
-export PATH=/cvmfs/cms.cern.ch/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin:/cvmfs/cms.cern.ch/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre3/external/el8_amd64_gcc12/bin:$PATH
-export LD_LIBRARY_PATH=/cvmfs/cms.cern.ch/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre3/biglib/el8_amd64_gcc12:/cvmfs/cms.cern.ch/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre3/lib/el8_amd64_gcc12:/cvmfs/cms.cern.ch/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre3/external/el8_amd64_gcc12/lib:/cvmfs/cms.cern.ch/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib64:/cvmfs/cms.cern.ch/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib:$LD_LIBRARY_PATH
-
-# These are the lines that you need to manually change for a CVMFS-less setup.
-# In this example we use cvmfs paths since that is where the dependencies are in lnx7188/cgpu1, but they can point to local directories.
-export BOOST_ROOT=/cvmfs/cms.cern.ch/el8_amd64_gcc12/external/boost/1.80.0-60a217837b5db1cff00c7d88ec42f53a
-export ALPAKA_ROOT=/cvmfs/cms.cern.ch/el8_amd64_gcc12/external/alpaka/1.1.0-7d0324257db47fde2d27987e7ff98fb4
-export CUDA_HOME=/cvmfs/cms.cern.ch/el8_amd64_gcc12/external/cuda/12.4.1-06cde0cd9f95a73a1ea05c8535f60bde
-export ROOT_ROOT=/cvmfs/cms.cern.ch/el8_amd64_gcc12/lcg/root/6.30.07-21947a33e64ceb827a089697ad72e468
-export CMSSW_BASE=/cvmfs/cms.cern.ch/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre3
-
-# These lines are needed to account for some extra environment variables that are exported in the setup script.
-export LD_LIBRARY_PATH=$PWD/SDL/cuda:$PWD/SDL/cpu:$PWD:$LD_LIBRARY_PATH
-export PATH=$PWD/bin:$PATH
-export PATH=$PWD/efficiency/bin:$PATH
-export PATH=$PWD/efficiency/python:$PATH
-export TRACKLOOPERDIR=$PWD
-export TRACKINGNTUPLEDIR=/data2/segmentlinking/CMSSW_12_2_0_pre2/
-export LSTOUTPUTDIR=.
-source $PWD/code/rooutil/thisrooutil.sh
-
-# After this, you can compile and run LST as usual.
-sdl_run -f -mc -s PU200 -n -1 -t myTag
+scram b -j 12 code-checks >& c.log && scram b -j 12 code-format >& f.log
 ```
 
-## Code formatting and checking
-
-The makefile in the `SDL` directory includes phony targets to run `clang-format` and `clang-tidy` on the code using the formatting and checks used in CMSSW. The following are the available commands.
+In any case, the makefile in the `standalone` directory also includes phony targets to run `clang-format` and `clang-tidy` on the code using the formatting and checks used in CMSSW. The following are the available commands.
```

Comment:
I think these makefile targets probably don't work anymore since they assume that files are in the SDL directory. I think it's probably not worth fixing them and just switch to using scram.

Reply:
Ok, deleting them then.